Dockerfile

```diff
@@ -7,9 +7,16 @@ COPY . /opt/app
 WORKDIR /opt/app/fact_extractor
 
 ARG USER=root
+ARG DEBIAN_FRONTEND=noninteractive
 
-RUN install/pre_install.sh
+RUN install_clean python3.11 python3.11-dev python3.11-venv gcc
```

Comment:
Originally I added this to fix compatibility issues with existing apt python packages. Since I also fixed it for the CI afterwards (without venv) it could be no longer necessary. It should be generally more robust, though, so we might want to keep it.